libs/packages/components/src/lib/search-result-list/search-result-list.component.html

```diff
@@ -5,5 +5,15 @@
 </div>
 <hr *ngIf="divider" class="thin" />
 <div *ngIf="!model|| model.length===0">
-    No results found
+    <div class="grid-row border-1px border-base-lighter" style="background-color:#F7F9FA;">
```

Comment:
Remove style and replace with bg-gray-cool-2 class

Reply:
that doesn't work, the system color tokens from uswds aren't imported so I used an inline style instead